src/els_compiler_diagnostics.erl

```diff
@@ -241,7 +245,9 @@ include_lib_id(Path) ->
   Length     = length(Components),
   End        = Length - 1,
   Beginning  = max(1, Length - 2),
-  filename:join(lists:sublist(Components, Beginning, End)).
+  [H|T]      = lists:sublist(Components, Beginning, End),
+  %% Strip the app version number from the path
```

Comment:
Not stripping the eventual app number was causing the warning to be displayed on the first line of the module (fallback).